notebooks/curation_api/R/get_dataset_assets_R.ipynb

```diff
@@ -3,25 +3,42 @@
   {
    "cell_type": "markdown",
    "id": "c5490d93",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
     "# Download a Dataset's available assets"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "74560383",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
     "The script in this notebook retrieves download links for a Dataset and then uses those links to download all available assets.\n",
     "\n",
-    "Fetching a Dataset requires only the Collection id and the Dataset id; it does not require an API key/access token."
+    "Fetching a Dataset requires only the Collection id and the Dataset id; it does not require an API key/access token.\n",
+    "\n",
+    "Note: This notebook guides download for assets of the current version of a dataset. For public collections, that is the most recently\n",
+    "published version of a dataset. For private collections, that is the most recently successfully processed dataset version. This notebook does not cover downloading previously published versions of a revised dataset.\n",
```

Comment:
> This notebook does not cover downloading previously published versions...

I'm a little hesitant to have wording like this. The download process is identical for _datasets_ and _dataset_versions_. But this wording suggests that they are markedly different.

Reply:
I think this portion should just be removed; leave the user to port the logic here to the dataset objects returned on the `.._versions/..` endpoints.

Reply:
I think that we need to be clear that a dataset url is a permanent url for **a** version of a dataset.

Reply:
I can remove that line, fair enough that it can be confusing. I do want to keep the rest since Brian wanted us to be clear with users that they are downloading a version of a dataset. So I want to be clear about what exactly they are downloading in the example scenario since it does slightly differ depending on whether the collection is public or private. 

> This notebook demonstrates downloading assets of the current version of a particular dataset. For public collections, that refers to the most recently published version of a dataset. For private collections, that refers to the most recently successfully processed dataset version.

Reply:
That's fine!

Reply:
Also agree with Brian's comment about specifying that it is a permalink--for that specific version.